code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -229,17 +231,28 @@ can cause issues with ammo types getting mixed up during the burst.
 	flags_gun_features = GUN_CAN_POINTBLANK|GUN_INTERNAL_MAG|GUN_SHOTGUN_CHAMBER|GUN_AMMO_COUNTER
 	current_mag = /obj/item/ammo_magazine/internal/shotgun/combat
 	attachable_allowed = list(
-						/obj/item/attachable/bayonet,
-						/obj/item/attachable/reddot,
-						/obj/item/attachable/flashlight,
-						/obj/item/attachable/extended_barrel,
-						/obj/item/attachable/compensator,
-						/obj/item/attachable/magnetic_harness,
-						/obj/item/attachable/stock/tactical)
+		/obj/item/attachable/angledgrip,
+		/obj/item/attachable/attached_gun/flamer,
+		/obj/item/attachable/attached_gun/grenade,
+		/obj/item/attachable/attached_gun/shotgun, //Yo dawg I heard you like shotguns, so we put a shotgun on your shotgun.
```

Comment:
This attachment is currently bugged.
I see you like adding attachments to guns. Have you tested them to see if the sprites match?

Reply:
I tested it and it was fine.

Reply:
Did you test all attachments? And the sprites matched fine? Because last time we did it, they didn't.

Reply:
> Did you test all attachments? And the sprites matched fine? Because last time we did it, they didn't.

I tested all attachments. They weren't floating or anything.

Reply:
How did you manage to test this attachment, when the gun cannot hold it because it has an unremovable UGL which is not removed in this code?